azure-kusto-ingest/azure/kusto/ingest/_resource_manager.py

```diff
@@ -63,6 +64,9 @@ def is_applicable(self):
 
 
 class _ResourceManager:
+    _SHOW_VERSION = ".show version"
```

Comment:
you can make those function locals.
No need to add them at the class level.

Reply:
Agree it's unnecessary at this point, but I think this is a healthier design for reuse. I often see constants that tend to be reused  (like prefixes, queries, column names, error messages,, etc) defined multiple times, and I think that defining them to be accessible in the first place helps prevent that.

WDYT?

Reply:
1. Since python is not really private, it's odd adding this to the class.
2. The use case for reusability is slim, and if it exists, I'd reuse the whole function, not the constant.
3. If I'm not mistaken, with python this would be defined once even if it is function local.

Reply:
1. Python does have scoping though, so if it was defined only locally to the method it wouldn't be visible elsewhere. The only other option besides a refactor of a specialized file/class for all commands would be to **move it to the file level**. I do want this command to be reusable, so **I'll do that**, then.
I suspect it's the same rationale that was used for defining the _URI_FORMAT at the top of this same file - it's currently once used once in this file, but it can conceivably be used elsewhere, and it would be a shame to do anything but discourage duplicating it.
2. I don't agree - the very next use case is adding the feature to the Streaming client, where this same command will be necessary, but it'd have to be executed differently (can't reuse the method).
3. If it was function local, the value wouldn't exist until that function was called, which isn't guaranteed in valid use cases (see #2).